mod-source-record-manager-server/src/main/java/org/folio/dao/EventProcessedDao.java

```diff
@@ -7,12 +7,49 @@
 public interface EventProcessedDao {
 
   /**
-   * Saves eventId and handlerId to database
+   * This method saves eventId and handlerId to deduplication table.
    *
    * @param handlerId handler id
    * @param eventId event id
    * @param tenantId tenant id
-   * @return future if the event not processed yet
+   * @return successful future if event has not been processed already, or failed future otherwise
```

Comment:
@SerhiiNosko Do you mean vice versa? successful if the event has been processed?

Reply:
Yes, its correct, basically its deduplication table, if event already processed - it should return failed future with constraint violation exception, and our service recognized it and throws DuplicateEventException.
If event has not been yet processed - event id should be inserted in this table, and successful future will be returned

Reply:
Updated documentations https://wiki.folio.org/pages/viewpage.action?pageId=59932680. Here is what was changed:

- adjusted default values for consumer's count

![image](https://user-images.githubusercontent.com/25097693/173087980-28a77a0a-c842-4939-91ab-2b93a732f521.png)

- Add paragraph regarding topic's partition configuration
![image](https://user-images.githubusercontent.com/25097693/173088203-5d33a115-8d81-49f6-814d-a69b362cd327.png)